WooCommerce/Classes/Authentication/Epilogue/StorePickerViewController.swift

```diff
@@ -244,13 +261,15 @@ private extension StorePickerViewController {
         tableView.registerNib(for: EmptyStoresTableViewCell.self)
         tableView.registerNib(for: StoreTableViewCell.self)
         tableView.backgroundColor = backgroundColor()
+        tableView.sectionFooterHeight = 0
     }
 
     func setupAccountHeader() {
         guard let defaultAccount = ServiceLocator.stores.sessionManager.defaultAccount else {
             return
         }
 
+        accountHeaderView.updateContainerView(hasBorders: isSimplifiedLogin)
```

Comment:
@itsmeichigo I see you are not adding the borders to the user info part when simplified login is not enabled.

I'm doing it for all cases, I did this because this the sense that I got from discussing this with Joe p1666708196780059-slack-C046HDZL87J, WDYT? Should we confirm this point too?

Reply:
Good point 👍 I asked Joe on the Figma design ERGtCFLyiC9CFpHsw1XIid-fi-392%3A3599.

Reply:
He replied that we should apply this to all variants so I'll update it on my PR.